appeals/web/src/server/appeals/appeal-details/interested-party-comments/add-ip-comment/add-ip-comment.controller.js

```diff
@@ -217,7 +220,65 @@ export async function postDateSubmitted(request, response) {
  * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
  */
 export async function postIPComment(request, response) {
-	response.end();
+	const documentGuid = request.session.fileUploadInfo?.files[0].GUID
+	const currentFolder = request.session.fileUploadInfo?.folderId
```

Comment:
It's probably a good idea to rename this, as by convention, `currentFolder` typically refers to a FolderInfo object rather than just the folder id, so this name could potentially cause some confusion. It could just be `folderId`?